tests/python/op/test_nn.py

```diff
@@ -86,5 +86,23 @@ def forward(self, x):
     verify_step(Model(), [x])
 
 
+@pytest.mark.parametrize("shape", [(3, 3, 3)])
```

Comment:
2 dimension case should work right? Can we add a test here? 
And also can you add a fp16 dtype test

Reply:
The test case permutes over the shapes. So basically it will do (1x1, 1x2, 1x3, 2x1,2x2...3x3). I tried running fp16, but bmm does not support it so it fails.